src/meshes/mesh_io.jl

```diff
@@ -249,10 +250,10 @@ function load_mesh_serial(mesh_file::AbstractString; n_cells_max, RealT)
     end
 
     if mesh_type == "TreeMesh"
-        n_cells = h5open(mesh_file, "r") do file
-            return read(attributes(file)["n_cells"])
+        capacity = h5open(mesh_file, "r") do file
+            return read(file["capacity"])
```

Comment:
```suggestion
            return read(attributes(file)["capacity"])
```
With the change above this would need to read an attribute, not a variable